deploy/helm/flightctl/charts/ui/templates/flightctl-ui-cm.yaml

```diff
@@ -28,9 +28,7 @@ data:
   BASE_UI_URL: {{ include "flightctl.getUIUrl" . }}
   {{- end }}
   {{- if eq (.Values.global).target "standalone" }}
-  {{- if eq ((.Values.global).auth).type "builtin" }}
-  INTERNAL_AUTH_URL: http://keycloak:8081/realms/flightctl
-  {{- else if eq ((.Values.global).auth).type "oidc" }}
+  {{- if eq ((.Values.global).auth).type "oidc" }}
```

Comment:
I will create a PR to drop the `INTERNAL_AUTH_URL` and `AUTH_CLIENT_ID` since these are for the current way of having a single provider.